dnf/cli/commands/module.py

```diff
@@ -271,6 +271,28 @@ def run_on_module(self):
 
             logger.error(dnf.exceptions.MarkingErrors(no_match_group_specs=skipped_groups))
 
+    class SwitchSubCommand(SubCommand):
+
+        aliases = ('switch',)
+        summary = _('switch module stream and and distrosync rpm packages')
+
+        def configure(self):
+            demands = self.cli.demands
+            demands.available_repos = True
+            demands.sack_activation = True
+            demands.resolving = True
+            demands.root_user = True
+            self.base.conf.module_stream_switch = True
+
+        def run_on_module(self):
+            try:
+                self.module_base.switch(self.opts.module_spec, strict=self.base.conf.strict)
+            except dnf.exceptions.MarkingErrors as e:
+                if self.base.conf.strict:
+                    if e.no_match_group_specs or e.error_group_specs:
+                        raise e
+                logger.error(str(e))
```

Comment:
This a meaning of a general configuration conf.strict or --skip-broken:
```
``--skip-broken``
    Resolve depsolve problems by removing packages that are causing problems from the transaction.
    It is an alias for the :ref:`strict <strict-label>` configuration option with value ``False``.
    Additionally, with the :ref:`enable <module_enable_command-label>` and
    :ref:`disable <module_disable_command-label>` module subcommands it allows one to perform an action even in case of
    broken modular dependencies.
```
```
``strict``
    :ref:`boolean <boolean-label>`

    If disabled, all unavailable packages or packages with broken dependencies given to DNF command will be skipped without raising the error causing the whole operation to fail. Currently works for install command only. The default is True.
```

Reply:
Right, I just didn't read the code well enough, sorry.